data/dirwolf.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Dhruv rawat",
+  "bio": "Computer science undergrad",
+  "type": "personal"
+  "avatar": "https://avatars.githubusercontent.com/u/92151410?v=4"
+  "links": [
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/dhruv-rawat-aa8305220/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCuLxLMEyO8NambbHWwA5rew",
+      "icon": "FaYoutube"
+    },
+      {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/_dhruv.rawat_/",
+      "icon": "FaInstagram"
+    },
```

```suggestion
    }
```
The trailing comma isnot required.